### ISSUE TITLE
latest_run symlink created with --log_dir set

### ISSUE THREAD

**Issue by fredrik-hammar-dd** [hawk/lux#11]
Lux creates a latest_run link when using --log_dir, even though the documentation states it does not do that. With version 1.13.2.

> If the configuration parameter --log_dir LogDir is set, the given path will be used instead and no symbolic link will be created.

```
$ lux --log_dir foo/bar
summary log       : /home/frehamma/work/ntte-aio/src/ntte-aio/test/lux/tailf-hcc/foo/bar/lux_summary.log

successful        : 0
summary           : WARNING

file:///home/frehamma/work/ntte-aio/src/ntte-aio/test/lux/tailf-hcc/foo/bar/lux_summary.log.html
$ ls -l foo
total 4
drwxr-xr-x 1 frehamma frehamma 140 May  3 21:37 bar
lrwxrwxrwx 1 frehamma frehamma   3 May  3 21:37 latest_run -> bar
$ 
```

**Comment by hawk** [comment]
Thanks!

It is fixed now.

/Håkan

**Comment by fredrik-hammar-dd** [comment]
I have confirmed this is fixed in 1.14.1. Thanks!
